src/Nowruz/modules/credentials/creditStatus/index.tsx

```diff
@@ -1,15 +1,32 @@
 import { Icon } from 'src/Nowruz/general/Icon';
-
+import variables from 'src/components/_exports.module.scss';
 import css from './creditStatus.module.scss';
 import { statusType } from './status.types';
+import { Chip } from '../../general/components/Chip';
 
 export const CreditStatus = (props: statusType) => {
+  const { theme, icon, label } = props;
+  const iconColoe =
```

Comment:
create an object for it and typo on `iconColoe`
`
iconColor = {
'primary': variables.color_primary_500,
'success': variables.color_success_500,
...
`

usage: `iconColor={iconColor[theme]}`